packages/SwingSet/src/types-external.js

```diff
@@ -93,6 +93,7 @@ export {};
  * }} Message
  *
  * @typedef { 'none' | 'ignore' | 'logAlways' | 'logFailure' | 'panic' } ResolutionPolicy
+ * @typedef {{ name: string, upgradeMessage: string, seqNum: number }} DisconnectObject
```

Comment:
as discussed, `seqNum` feels a little terse and lacking context here: someone looking at a DisconnectObject in the resolution of `E(facetAdmin).upgrade(..)` will know that it has something to do with upgrade, and that it's a sequence of *something*, but the name doesn't suggest that we're talking about a sequence of vat incarnations, coupled by upgrade events. If I was willing to commit `incarnation` for this concept, then `incarnationNum` might provide enough context to make me happy, but I'm not 100% there yet.